src/main/java/org/folio/rest/impl/BLUsersAPI.java

```diff
@@ -104,6 +107,9 @@ public class BLUsersAPI implements BlUsers {
 
   private static final Pattern HOST_PORT_PATTERN = Pattern.compile("https?://([^:/]+)(?::?(\\d+)?)");
 
+  private static final String LOGIN_ENDPOINT = "/authn/login-with-expiry";
+  private static final String LOGIN_ENDPOINT_LEGACY = "/authn/login";
```

Comment:
These constants seem to be only used in a single place. Could they be moved to be local to that scope?